packages/css-library/tokens/line-lengths.json

```diff
@@ -0,0 +1,15 @@
+{
+  "vads": {
+    "size": {
+      "line-length": {
+        "1": { "value": "44ex" },
+        "2": { "value": "60ex" },
+        "3": { "value": "64ex" },
+        "4": { "value": "68ex" },
+        "5": { "value": "72ex" },
+        "6": { "value": "88ex" },
+        "none": { "value": "999ex" }
```

Comment:
Is this value `999ex` for `vads-size-line-length-none` taken from somewhere? Is that how USWDS is doing it? I'm looking at [our docs](https://design.va.gov/foundation/typography#typography-tokens) and it just says "none".

Reply:
I tried putting in "none" as the value and Style Dictionary spit back that it wasn't a number, so I chose an arbitrarily long length to "simulate" a "none" length.

Reply:
@humancompanion-usds Do we need to support a "none" line-length? Is making it arbitrarily long the right method to take given the limitations of Style Dictionary? I'm thinking that maybe just not having this token is sufficient.

Reply:
@Andrew565 Maybe a value with a similar affect that style dictionary will accept is `unset`, `inherit`, `initial`, or `revert`:

https://css-tricks.com/inherit-initial-unset-revert/

Reply:
Yes we should have a none token and yes "inherit" should probably be the value.